### ISSUE TITLE
Create skipped jobs

### ISSUE THREAD

**Issue by altany** [travis-ci/docs-travis-ci-com#2991]
Is there a way to include a job to my configuration but skip it.

Meaning that I want to have the option to run the job manually if I chose to - for example by clicking the restart button, as I would if I had manually cancelled -  and not have it run automatically.

This is different to adding conditionals in a stage as the idea is to have a job available for all relevant builds, but I get to chose if it runs.

**Comment by kasperk81** [comment]
I don't think it is currently possible on TravisCI. I have used it on GitHub Actions [workflow dispatch](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/).

here is a quick example: https://github.com/kasperk81/actionstesting/actions/workflows/myoptionalflow.yml
with regular merge to master, PR or tag push, it will skip the optional job:
![2021-07-15_285x169_scrot](https://user-images.githubusercontent.com/83082615/125695462-0e3ce5dd-a37f-426d-8c51-fde92bdd79c3.png)
and to manually run the optional job, the button will show up to the owner (or person who has access, collaborator/member etc.):
![2021-07-15_1278x476_scrot](https://user-images.githubusercontent.com/83082615/125695801-d023c932-fd5b-4036-b6fb-81d0e18d0bfe.png)
change "nope" to "yes" and hit Run workflow, it will then run the optional step:
![2021-07-15_275x165_scrot](https://user-images.githubusercontent.com/83082615/125695853-7c298c28-5a56-49a8-a6b9-e345c777a49b.png)
(you can fork and try)
Note: here you are in control of "optional' or "skip" semantics, whichever way you want to construct your workflow. Also optional and default jobs can be separated on workflow level too, i.e. in separate workflow yml files.
